app/forms/fields.py

```diff
@@ -168,7 +168,11 @@ def get_number_field(answer, label, guidance, error_messages, answer_store):
 
     mandatory_or_optional = get_mandatory_validator(answer, answer_errors)
 
-    max_decimals = answer.get('decimal_places', 0)
+    if answer['type'] == "Currency":
+        max_decimals = 2
```

Comment:
Defaulting currency to two decimal places should be removed until agreed by all business areas as this is not the behaviour of production. When rolling out the use of pence within the currency we can add the "decimal_places": 2 to the schema.